dive_core/gfxr_vulkan_command_hierarchy.cpp

```diff
@@ -0,0 +1,263 @@
+/*
+ Copyright 2025 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+#include "gfxr_vulkan_command_hierarchy.h"
+#include "dive_strings.h"
+
+namespace Dive
+{
+GfxrVulkanCommandHierarchyCreator::GfxrVulkanCommandHierarchyCreator() {}
```

Comment:
better to add "//--------------------------------------------------------------------------------------------------" after this func to separate functions from each other.